src/main/common/interestUtils.ts

```diff
@@ -13,13 +13,19 @@ import { InterestType as ClaimInterestType } from 'claims/models/interestType'
 import { YesNoOption } from 'models/yesNoOption'
 import { InterestTypeOption } from 'claim/form/models/interestType'
 
+function getInterestEndDate (issuedOn: moment.Moment, interestFromDate: moment.Moment): moment.Moment {
+  const currentDate = MomentFactory.currentDate()
+  const endDate = issuedOn > currentDate ? issuedOn : currentDate
```

Comment:
Why do we need this hack? Cannot we just use issueDate?

Reply:
no, this fix is for  weekend issue, 
if claims is issued after 4 PM on Friday, then date should be displayed as issue date, not current date and after weekend it will be current date

Reply:
But if it's not a weekend then current date is issue date.

Reply:
The other thread clarified it a bit for me. Still, the end date in that case should be a `max` between current date and issue date. We shouldn't be interest in from date at all.